NetCore/project.json

```diff
@@ -5,18 +5,19 @@
     "emitEntryPoint": true
   },
   "frameworks": {
-    "netcoreapp1.0": {
+    "netcoreapp1.1": {
```

Comment:
By the way, this is the output from my build on my quad-core i7 laptop.  Both the speed and quality are better than the other libraries you tested once you get BenchmarkDotNet to run it...

|                           Method |        Mean |    StdDev |      Gen 0 |     Gen 1 |     Gen 2 | Allocated |
|--------------------------------- |------------ |---------- |----------- |---------- |---------- |---------- |
|  'ImageSharp Load, Resize, Save' | 811.3793 ms | 1.9278 ms | 10158.3333 | 9512.5000 | 9208.3333 |   92.8 MB |
| 'ImageMagick Load, Resize, Save' | 746.6892 ms | 6.9822 ms |          - |         - |         - |  60.27 kB |
|   'ImageFree Load, Resize, Save' | 491.3403 ms | 0.8399 ms |  6033.3333 |   54.1667 |         - | 122.55 kB |
| 'MagicScaler Load, Resize, Save' | 167.4302 ms | 0.8310 ms |   175.0000 |         - |         - |   3.66 MB |

![dscn0533-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402761/55adb8aa-e5b7-11e6-9dc9-d1dc88f3234e.JPG) ![img_2301-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402763/55af1a42-e5b7-11e6-841a-1302eba22c5f.jpg) ![img_2317-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402762/55add7e0-e5b7-11e6-87ee-7e9fa3b2ba59.jpg) ![img_2325-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402764/55af6f4c-e5b7-11e6-9b84-60b28ff0029a.jpg) ![img_2351-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402766/55b00af6-e5b7-11e6-9aee-e2a3c3ba6ec9.jpg) ![img_2443-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402765/55afcf78-e5b7-11e6-8d64-d36744f43cae.jpg) ![img_2445-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402767/55bcb346-e5b7-11e6-814a-dc3598247033.jpg) ![img_2446-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402768/55bcb738-e5b7-11e6-8938-2f6fc4f32009.jpg) ![img_2525-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402769/55becf8c-e5b7-11e6-9173-e73affcfb361.jpg) ![img_2565-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402771/55c02fe4-e5b7-11e6-8362-272b2fdf461f.jpg) ![img_2734-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402770/55bf7fcc-e5b7-11e6-96ba-6cc6270689c8.jpg) ![sample-magicscaler](https://cloud.githubusercontent.com/assets/7929102/22402772/55c5a8ac-e5b7-11e6-8b46-b7ee7d0e023e.jpg)